src/bootstrap/bootstrap.py

```diff
@@ -16,12 +16,26 @@
 import subprocess
 import sys
 import tarfile
+import tempfile
+
 
 def get(url, path, verbose=False):
-    print("downloading " + url)
     sha_url = url + ".sha256"
-    sha_path = path + ".sha256"
-    for _url, _path in ((url, path), (sha_url, sha_path)):
+    temp_file = tempfile.NamedTemporaryFile(delete=False)
+    temp_path = temp_file.name
+    sha_file = tempfile.NamedTemporaryFile(suffix=".sha256", delete=True)
+    sha_path = sha_file.name
+    download(sha_path, sha_url, temp_path, url, verbose)
```

Comment:
I think it might be worth reducing number of parameters here and invoking `download` twice instead, to show that we indeed download 2 files here ?